### DISCUSSION TITLE
Contribution detail page layout

### DISCUSSION THREAD

**Discussion by Depechie** [sthewissen/MVP#108] [General]
Just wondering why the Center was chosen?  https://github.com/sthewissen/MVP/blob/main/MVP/Pages/ContributionDetailsPage.xaml#L54

Now when you have a small contribution it will be centered, a large contribution will be top aligned and scrollable.

**Comment by sthewissen** [comment]
No particular reason actually. Can safely make it `StartAndExpand` I think.

**Reply by Depechie** [reply]
![image](https://user-images.githubusercontent.com/351693/123435705-24a98200-d5ce-11eb-9f88-94b09be8f510.jpeg)

For reference a contrib that is small and centred.
